src/server/blockchain/AdminWallet.js

```diff
@@ -289,21 +294,20 @@ export class Wallet {
   /**
    * whitelist an user in the `Identity` contract
    * @param {string} address
-   * @param {string} did
    * @returns {Promise<TransactionReceipt>}
    */
-  async whitelistUser(address: string, did: string): Promise<TransactionReceipt | boolean> {
+  async whitelistUser(address: string): Promise<TransactionReceipt | boolean> {
```

Comment:
that's a mistake did should not have been removed... we need to fix the adminwallet contracts

Reply:
done in https://github.com/GoodDollar/GoodServer/pull/139/commits/113e7728f6d745c5e872c0ddfb275e9e9390919a